### ISSUE TITLE
Python problem on Ubuntu 18.04

### ISSUE THREAD

**Issue by stoiver** [SGpp/SGpp#109]
Just installed the latest version of SGpp on Unbuntu 18.04 on Windows.

On testing the python bindings I get the following error 

```
anuga@DESKTOP-QB0OQBJ:~$ ipython3
Python 3.6.7 (default, Oct 22 2018, 11:32:17)
Type "copyright", "credits" or "license" for more information.

IPython 5.5.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import pysgpp
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-d02860db5749> in <module>()
----> 1 import pysgpp

/home/anuga/SGpp/lib/pysgpp/__init__.py in <module>()
      6
      7 # import pysgpp_swig and extensions
----> 8 from pysgpp_swig import *
      9 import pysgpp.extensions

/home/anuga/SGpp/lib/pysgpp/pysgpp_swig.py in <module>()
      5 # the SWIG interface file instead.
      6
----> 7 from . import _pysgpp_swig
      8 try:
      9     _swig_property = property

ImportError: attempted relative import with no known parent package
```

The error seemed to be fixed by changing line 7 of `pysgpp_swig.py` from 
```
from . import _pysgpp_swig
```
to
```
import _pysgpp_swig
```

**Comment by spc90** [comment]
@stoiver Thank you for raising this issue. I have not been able yet to duplicate this with a clean clone on Ubuntu 18.04 on a Linux machine. 

What I can see from the ipython trace is that you have a different `lib/pysgpp/__init__.py` content than what I would expect to be generated. If this is not what the build generates for you then indeed this is a problem. I would have expected the imports to normally read
```
from .pysgpp_swig import *
from . import extensions
```
Could you please provide the full content of that file after building SG++?

**Comment by valentjn** [comment]
Yeah, I think this is a version problem of SG++ or SWIG. My guess is that it's SG++.

For the current version of SG++, `lib/pysgpp/__init__.py` should be the two lines that @spc90 wrote (and not `from pysgpp_swig import *` as @stoiver's traceback). In addition, the `from . import _pysgpp_swig` line has been removed from the `pysgpp/pysgpp.i` file (which generates `lib/pysgpp/pysgpp_swig.py`) in commit 92b6b7b2, which was merged to `master` in #91 (releases v3.2.0 and above). This indicates that the installed version of SG++ is probably too old.

**Comment by yashsharan** [comment]
Has this issue been fixed? Even I am getting the exact same error when I try to run import pysgpp.

**Comment by leiterrl** [comment]
> Has this issue been fixed? Even I am getting the exact same error when I try to run import pysgpp.

As the above discussion indicates this issue should be fixed in v3.2.0. At least we could not reproduce it on a fresh install.

If the issue persists for you it would help if you could provide some more information about your setup such as OS, SG++ version etc.
